packages/loader/container-loader/src/connectionManager.ts

```diff
@@ -245,6 +245,10 @@ export class ConnectionManager implements IConnectionManager {
 
 	public readonly clientDetails: IClientDetails;
 
+	public get supportedFeatures(): Record<string, unknown> | undefined {
```

Comment:
These are not supportedFeatures of connectionManager, so should probably rename it to: "relayServiceSupportedFeatures", relay service is the ordering service.